integration-cli/docker_utils.go

```diff
@@ -782,13 +782,13 @@ func dockerCmdInDirWithTimeout(timeout time.Duration, path string, args ...strin
 	return integration.DockerCmdInDirWithTimeout(dockerBinary, timeout, path, args...)
 }
 
-func findContainerIP(c *check.C, id string, vargs ...string) string {
-	out, _ := dockerCmd(c, "inspect", "--format='{{ .NetworkSettings.IPAddress }}'", id)
+func findContainerIP(c *check.C, id string, network string) string {
+	out, _ := dockerCmd(c, "inspect", fmt.Sprintf("--format='{{ .NetworkSettings.Networks.%s.IPAddress }}'", network), id)
 	return strings.Trim(out, " \r\n'")
 }
 
 func (d *Daemon) findContainerIP(id string) string {
-	return findContainerIP(d.c, id, "--host", d.sock())
+	return findContainerIP(d.c, id, "--host")
```

Comment:
@thaJeztah   thanks
@tiborvass   After investigating why `return findContainerIP(d.c, id, "--host")` unexpectedly make CI green and found that we can't call `findContainerIP` in ` func (d *Daemon) findContainerIP(id string)` because `findContainerIP` connect to the daemon start by `.integration-daemon-start`, we should connect the daemon `d` here, so I think the correct fix here is
<pre><code>func (d *Daemon) findContainerIP(id string) string {
        out, _ := d.Cmd("inspect", fmt.Sprintf("--format='{{ .NetworkSettings.Networks.bridge.IPAddress }}'"), id)
        return strings.Trim(out, " \r\n'")
}
</code></pre>
correct me if I'm wrong :-)